front/src/components/boxs/device-in-room/EditDeviceInRoom.jsx

```diff
@@ -39,8 +39,8 @@ class EditDeviceInRoom extends Component {
             value: feature.selector,
             label: getDeviceFeatureName(this.context.intl.dictionary, device, feature)
           };
-          // for now, we only supports binary on/off and sensors
-          if (feature.read_only || feature.type === DEVICE_FEATURE_TYPES.LIGHT.BINARY) {
+          // for now, we only supports binary on/off, brightness and sensors
+          if (feature.read_only || [DEVICE_FEATURE_TYPES.LIGHT.BINARY, DEVICE_FEATURE_TYPES.LIGHT.BRIGHTNESS].includes(feature.type)) {
```

Comment:
Can we extract this in a const a file top level? For more readability and reusability.

Reply:
Done with rebase